modules/mono/glue/GodotSharp/GodotSharp/Core/Attributes/GodotClassNameAttribute.cs

```diff
@@ -16,6 +16,10 @@ public class GodotClassNameAttribute : Attribute
         /// </summary>
         public string Name { get; }
 
+        /// <summary>
+        /// Specify the name that represents the original engine class.
+        /// </summary>
+        /// <param name="name">Name of the original engine class.</param>
         public GodotClassNameAttribute(string name)
```

Comment:
We should consider hiding this attribute with `[EditorBrowsable(Never)]`. It probably should have been `internal` but now we can't change that without breaking compat. Users tend to confuse it with the `[GlobalClass]` attribute.